scripts/fuzz_opt.py

```diff
@@ -319,7 +351,9 @@ def test_one(random_input, opts):
   shutil.copyfile('a.js', 'b.js')
 
   for testcase_handler in testcase_handlers:
-    testcase_handler.handle_pair(before_wasm='a.wasm', after_wasm='b.wasm', opts=opts + FUZZ_OPTS + FEATURE_OPTS)
+    print('running testcase handler:', testcase_handler.__class__.__name__)
+    if testcase_handler.can_run_on_feature_opts(FEATURE_OPTS):
```

Comment:
Doesn't this do a lot of spinning, since not many features are actually supported by all the tools? It also would have seemed more natural to start with MVP and add additional features rather than selectively removing features. Perhaps I just don't understand the motivation for this change, though.

Reply:
It still runs at least `FuzzExec` on any feature combination, so it doesn't spin idly.

It also has a 50% chance to add all the disablings, which will enable all the tools.

So this does run `FuzzExec` and other tools more often than before, it's true, but this way it actually tests those things on arbitrary combinations of features, instead of testing all things on the lowest common denominator - which meant we were just not testing many features at all.

Reply:
Aha, I hadn't appreciated that it will be testing MVP half the time. LGTM.